src/app/libp2p_helper/src/libp2p_helper/main.go

```diff
@@ -1211,6 +1214,44 @@ func (ap *findPeerMsg) run(app *app) (interface{}, error) {
 	return *maybePeer, nil
 }
 
+type setConsensusNodeMsg struct {
+	IsConsensusNode bool `json:"IsConsensusNode"`
+}
+
+func (cn *setConsensusNodeMsg) run(app *app) (interface{}, error) {
+	if app == nil {
+		return nil, needsConfigure()
+	}
+	app.P2p.IsConsensusNode = cn.IsConsensusNode
+	return "consensusNodeMsg set successfully", nil
+}
+
+type getConsensusNodeMsg struct {
+	PeerMultiaddr string `json:"peer_multiaddr"`
+}
+
+func (gcn *getConsensusNodeMsg) run(app *app) (interface{}, error) {
```

Comment:
not sure if this would be used, since the OCaml daemon would know whether it's consensus or not, it wouldn't need to get it from here